tests/routes/conftest.py

```diff
@@ -1,7 +1,155 @@
+        },
+        {
+            "Key Concept": "Position - Elements and the Coordinate System",
+            "Observations": "The program places two sprites and one line of text using the coordinate system. No shapes are created.",
+            "Evidence": "Lines 2-3: Two sprites are positioned using coordinates.\n`var animalhead_duck1 = createSprite(352, 200);\nvar animalhead_frog_1 = createSprite(46, 200);`\n\nLine 12: One line of text is positioned using coordinates.\n`text(\"Fortnite\", 175, 200);`",
+            "Reason": "The program demonstrates the use of the coordinate system to place elements on the screen. It positions two sprites (animalhead_duck1 and animalhead_frog_1) and one line of text (\"Fortnite\") using x and y coordinates. However, it does not create any shapes using functions like rect, ellipse, circle, quad, or triangle. According to the rubric, for Convincing Evidence, we need at least 1 shape, 2 sprites, and 1 line of text. While the program meets the criteria for sprites and text, it lacks any shapes. Therefore, it falls under the Limited Evidence category. Decision: Limited Evidence",
+            "Grade": "Limited Evidence"
+        }
+    ]
+
+def get_claude_response_data(eval_data):
+    eval_json = json.dumps(eval_data)
+    return {
+        "id": "msg_bdrk_01234567890abcdefghijklm",
+        "type": "message",
+        "role": "assistant",
+        "model": "claude-3-5-sonnet-20240620",
+        "content": [
+            {
+                "type": "text",
+                "text": eval_json
+            }
+        ],
+        "stop_reason": "end_turn",
+        "stop_sequence": None,
+        "usage": {
+            "input_tokens": 1397,
+            "output_tokens": 972
+        }
+    }
+
+def get_openai_response_data(eval_data):
+    return {
+        "id": "chatcmpl-8FpJ3JGggzw1sfU7V2zkGsTnmH8vn",
+        "object": "chat.completion",
+        "created": 1698782833,
+        "model": "gpt-4-0613",
+        "choices": [
+            {
+                "index": 0,
+                "message": {
+                    "role": "assistant",
+                    "content": json.dumps(eval_data)
+                },
+                "finish_reason": "stop"
+            }
+        ],
+        "usage": {
+            "prompt_tokens": 1798,
+            "completion_tokens": 1071,
+            "total_tokens": 2869
+        }
+    }
+
+@pytest.fixture
+def lesson_11_claude_response_body(lesson_11_eval):
+    response_data = get_claude_response_data(lesson_11_eval)
+    yield json.dumps(response_data)
+
+@pytest.fixture
+def lesson_11_claude_response_body_mismatched(lesson_11_eval):
+    eval = lesson_11_eval
+    eval[0]["Key Concept"] = "Bogus Key Concept"
+    response_data = get_claude_response_data(eval)
+    yield json.dumps(response_data)
+
+@pytest.fixture
+def lesson_11_claude_response_body_too_large():
+    unparsable_json = '["unparsable JSON"'
+    response_data = get_claude_response_data(unparsable_json)
```

Comment:
Wasn't immediately obvious why the response data would have unparsable json until I realized that it is meant to simulate the response truncating the json. Not sure if we want some documentation of that or to update the name of this fixture, but I think it is also fine as it is.